attendance/handle_query.py

```diff
@@ -1,4 +1,5 @@
 import day_functions
+import string_functions
 
 
```

Comment:
I see you are using the same functions that are used by the same name - get_response, process_query. Would it help if you write a base class that can be used as a template for all the other modules. It is not necessary for now as it would complicate things. But in future it would easy for contributors to follow up with other modules.

Reply:
each package has their own string_functions and handle_query py modules. they all have different functions in them. I'll change if i can.